frontend/automation/playwright.config.ts

```diff
@@ -41,59 +43,35 @@ export default defineConfig({
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
     trace: 'on-first-retry',
-    screenshot: 'only-on-failure'
+    screenshot: 'only-on-failure',
+    ...devices['Desktop Chrome'],
+    launchOptions: {
+      args: ['--disable-web-security']
+    },
+    permissions: ['clipboard-read']
   },
 
-  /* Configure projects for major browsers */
   projects: [
     {
-      name: 'chromium',
-      use: {
-        ...devices['Desktop Chrome'],
-        launchOptions: {
-          args: ['--disable-web-security']
-        },
-        permissions: ['clipboard-read']
-      }
+      name: 'default',
+      testIgnore: /.*fairground.test.ts/
+    },
+    {
+      name: 'fairground',
+      testMatch: /.*fairground.test.ts/,
+      retries: 0
     }
-
-    // {
-    //   name: 'firefox',
-    //   use: { ...devices['Desktop Firefox'] }
-    // }
-
-    // {
-    //   name: 'webkit',
-    //   use: { ...devices['Desktop Safari'] },
-    // },
-
-    /* Test against mobile viewports. */
-    // {
-    //   name: 'Mobile Chrome',
-    //   use: { ...devices['Pixel 5'] },
```

Comment:
are these mobile sizes needed here? since this is a desktop app, i think most of these can be just removed, and have like the OS specific defaults for macOS, linux and windows 🤔

Reply:
They were commented out, I just removed them

Reply:
tuesday's my colorblind day, my bad :)